code/modules/paperwork/faxmachine.dm

```diff
@@ -0,0 +1,321 @@
+	else
+		to_chat(usr, "There is nothing to remove from [src].")
+
+/obj/machinery/photocopier/faxmachine/proc/sendfax(var/destination,var/mob/sender)
+	if(stat & (BROKEN|NOPOWER))
+		return
+
+	use_power(200)
+
+	var/success = FALSE
+	for(var/thing in GLOB.allfaxes)
+		var/obj/machinery/photocopier/faxmachine/F = thing
+		if( F.department == destination )
+			success = F.receivefax(copy)
+	if(success != FALSE && department != destination)
+		var/datum/fax/F = new /datum/fax()
+		F.name = copy.name
+		F.from_department = department
+		F.to_department = destination
+		F.origin = src
+		F.message = copy
+		F.sent_by = sender
+		F.sent_at = world.time
+		visible_message("[src] beeps, \"Message transmitted successfully.\"")
+
+	else if(destination == department)
+		visible_message("[src] beeps, \"Error transmitting message. [src] cannot send faxes to itself.\"")
+	else if(destination == "Not Selected")
+		visible_message("[src] beeps, \"Error transmitting message. Select a destination.\"")
+	else if(destination == "Unknown")
+		visible_message("[src] beeps, \"Error transmitting message. Cannot transmit to Unknown.\"")
+	else
+		visible_message("[src] beeps, \"Error transmitting message.\"")
+
+/obj/machinery/photocopier/faxmachine/proc/receivefax(var/obj/item/incoming)
+	if(stat & (BROKEN|NOPOWER))
+		return FALSE
+
+	if(department == "Unknown" || department == destination)
+		return FALSE	//You can't send faxes to "Unknown" or yourself
+
+	flick("faxreceive", src)
+
+	playsound(loc, 'goon/sound/machinery/printer_dotmatrix.ogg', 50, 1)
+
+	//give the sprite some time to flick
+	addtimer(CALLBACK(src, .proc/handle_copying, incoming), 20)
+
+/obj/machinery/photocopier/faxmachine/proc/handle_copying(var/obj/item/incoming)
+	use_power(active_power_usage)
+	if(istype(incoming, /obj/item/paper))
+		copy(incoming)
+	else if(istype(incoming, /obj/item/photo))
+		photocopy(incoming)
+	else
+		return FALSE
+
+	return TRUE
+
+/obj/machinery/photocopier/faxmachine/proc/send_admin_fax(var/mob/sender, var/destination)
+	if(stat & (BROKEN|NOPOWER))
+		return
+
+	if(sendcooldown)
+		return
+
+	use_power(200)
+
+	var/obj/item/rcvdcopy
+	if(copy)
+		rcvdcopy = copy(copy)
+	else if(photocopy)
+		rcvdcopy = photocopy(photocopy)
+	else
+		visible_message("[src] beeps, \"Error transmitting message.\"")
+		return
+
+	rcvdcopy.loc = null //hopefully this shouldn't cause trouble
+
+	var/datum/fax/admin/A = new /datum/fax/admin()
+	A.name = rcvdcopy.name
+	A.from_department = department
+	A.to_department = destination
+	A.origin = src
+	A.message = rcvdcopy
+	A.sent_by = sender
+	A.sent_at = world.time
+
+	//message badmins that a fax has arrived
+	switch(destination)
+		if("Central Command")
+			message_admins(sender, "CENTCOM FAX", destination, rcvdcopy, "#006100")
+		if("Syndicate")
+			message_admins(sender, "SYNDICATE FAX", destination, rcvdcopy, "#DC143C")
+	sendcooldown = cooldown_time
+	visible_message("[src] beeps, \"Message transmitted successfully.\"")
```

Comment:
All chat sent to Goonchat needs to be marked up with span tags, this'll be really relevant in the future if we try to "dark mode" this window too.